src/lib/components/Cytoscape.react.js

```diff
@@ -953,6 +965,27 @@ Cytoscape.propTypes = {
          */
         target: PropTypes.object,
     }),
+
+    /**
+     * Dictionary specifying configuration options to overlay a leaflet map on top of Cytoscape. All
+     * configuration options are optional; provide an empty dictionary to use default options. Requires
+     * latitude and longitude properties to be included in the node data for positional information.
+     * Requires preset layout to be used. Requires extra layouts to be loaded.
+     */
+    leaflet: PropTypes.exact({
+        /** Specify a tile preset from Leaflet providers instead of a manual tileUrl.  See http://leaflet-extras.github.io/leaflet-providers/preview/  See also https://github.com/leaflet-extras/leaflet-providers */
+        preset: PropTypes.string,
+        /** Endpoint used by leaflet to fetch map tiles. */
```

Comment:
```suggestion
        /** Endpoint used by leaflet to fetch map tiles (if not preset). */
```

Or `(if not using provider)` if we change the name.